js_modules/dagster-ui/packages/ui-core/src/ui/BaseFilters/useFilter.tsx

```diff
@@ -26,28 +27,49 @@ export const FilterTag = ({
   iconName,
   label,
   onRemove,
+  style = 'default',
 }: {
   label: JSX.Element;
   iconName?: IconName;
-  onRemove: () => void;
-}) => (
-  <div>
-    <BaseTag
-      icon={iconName ? <Icon name={iconName} color={Colors.linkDefault()} /> : undefined}
-      rightIcon={
-        <div onClick={onRemove} style={{cursor: 'pointer'}} tabIndex={0}>
-          <Icon name="close" color={Colors.linkDefault()} />
-        </div>
-      }
-      label={label}
-      fillColor={Colors.backgroundBlue()}
-      textColor={Colors.linkDefault()}
-    />
-  </div>
-);
+  onRemove?: () => void;
+  style?: 'default' | 'light-blue';
```

Comment:
- Maybe call it `cyan` instead of `light-blue`?
- `style` implies a CSS style object -- I was actually a little confused looking at the `style` prop on `SetFilterActiveState`. Maybe `theme` or something?